bin/mink-zombie-server.js

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/node
+var net = require('net');
+var zombie = require('zombie');
+var Tough = require('zombie/node_modules/tough-cookie');
+var browser = null;
+var pointers = [];
+var buffer = '';
+var host = process.env.HOST || '127.0.0.1';
```

Comment:
The driver needs to know where to connect to NodeJS server. In case if driver haven't provided any ENV vars and NodeJS server used the defaults the driver would have no way of knowing what defaults NodeJS server have chosen. To make this work we need to duplicate NodeJS server env defaults in the driver code, which isn't good.

I'm proposing to throw an exception in case if any of required ENV vars is missing.